doc/nrf/ug_bt_mesh_model_config_app.rst

```diff
@@ -0,0 +1,68 @@
+.. _ug_bt_mesh_model_config_app:
+
+Configuring mesh models using the nRF Mesh mobile app
+#####################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth mesh :ref:`samples <samples>` use the nRF Mesh mobile app to perform provisioning and configuration.
+
+Binding a model to an application key
+*************************************
+
+Complete the following steps in the nRF Mesh app to bind a model to an application key:
```

Comment:
I'd like to add an extra sentence of context to each of these sections. Suggestion:

> Mesh models encrypt all their messages using application keys. Models that should communicate with each other must be bound to the same application key.